packages/core/src/utils/transformers.ts

```diff
@@ -30,6 +30,7 @@ export function MetadataTransformer() {
  */
 export function DateTransformer() {
   return Transform(({ value, type }) => {
+    if (value === undefined) return undefined
```

Comment:
Should this not throw an error if the value is required? Not too familiar with class-transformer. But it seems like this could cause bugs, right?

Reply:
AFAIK transformation is separate from validation. I think it first transforms and then if isOptional is not provided and you validate it will throw an error